plugins/modules/artifact.py

```diff
@@ -91,6 +91,8 @@ def main():
 
         sha256 = module.params["sha256"]
         if module.params["file"]:
+            if not os.path.exists(module.params["file"]):
+                raise Exception("File not found.")
```

Comment:
Can we raise a SqueezerException here. There is no need to show the stacktrace, as we already know what's going wrong.

Reply:
Thanks for the hint, I wasn't aware of the custom exception.

Reply:
np It's handled here: https://github.com/pulp/squeezer/blob/develop/plugins/module_utils/pulp.py#L79